.github/ubiquibot-config.yml

```diff
@@ -28,6 +28,15 @@ priority-labels:
     weight: 4
   - name: "Priority: 4 (Emergency)"
     weight: 5
+default-labels:
+  global:
+    - "Time: <1 Hour"
+    - "Priority: 0 (Normal)"
+    - "Test"
+  users:
+    pavlovcik:
+      - "Time: <1 Hour"
+      - "Priority: 0 (Normal)"
```

Comment:
Pretty cool you implemented this but I still fear that we'll need to remove this when we implement the database style config e.g. `/default-labels @pavlovcik "Time: <1 Hour" "Priority: 0 (Normal)"